src/test/java/com/burgstaller/okhttp/AuthenticationCacheInterceptorTest.java

```diff
@@ -60,7 +46,7 @@ public class AuthenticationCacheInterceptorTest {
     @Mock
     Proxy proxy;
 
-    @Before
+    @BeforeEach
```

Comment:
Mocks only need to be initialised once

Reply:
I've integrated in the last update the mockito-junit-jupiter dependency which simplify mockito use.